packages/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dataunions/client",
-    "version": "3.0.0-beta.5",
+    "version": "3.0.0-beta.8",
```

Comment:
`package-lock.json` requires an update if `package.json` is updated. `npm install -w packages/client` should do the trick.